psa_car_controller/web/view/api.py

```diff
@@ -51,6 +51,18 @@ def get_vehicle_info(vin):
     )
     return response
 
+@app.route('/get_doors_state/<string:vin>')
+def get_doors_state(vin):
+    doors_state = APP.myp.remote_client.doors_state.get(vin)
```

Comment:
can you document possible value for door_state ?

Reply:
Like this?
```
/**
    @return {'doors_opening_state': number[]; 'doors_locking_state': number}
*/
```

Have you an example of documentation? And where i set? Thank you